elpis/gui/src/components/Model/Train.js

```diff
@@ -61,6 +61,10 @@ class ModelTrain extends Component {
     downloadjs(log, "log.txt", "text/txt");
   };
 
+  handleTensorBoard = () => {
+    window.open(window.location.protocol + "//" + window.location.hostname + ":6006", "_blank").focus();
```

Comment:
writer's comment: as horrible as this may look, this seems to be the universally accepted best way of doing this...